### ISSUE TITLE
Refactoring: remove helpers.get_abspath()

### ISSUE THREAD

**Issue by truth-quark** [ACCESS-NRI/amami#42]
The `helpers.get_abspath()` function is not required once #40 is complete:

* `sys.exit()` calls from the custom logging module are being removed
* Refactoring to exceptions means `get_abspath()` throws `IOError` if files are missing (duplicating Python's functionality).
* The function is only used 2x

**Comment by atteggiani** [comment]
Hmm yes that is true, but the `helpers.get_abspath()` function might still be needed to get the path we want logged. 
I think Python's `IOError` does not necessarily output the absolute path, but whatever argument was given as a path.

**Comment by truth-quark** [comment]
Ooops, Python throws a `FileNotFound` error which shows the path (abs or relative, whichever is passed to the func). This is fine as `um2nc` doesn't change the working directory.

**Comment by atteggiani** [comment]
Yeah, I created the `get_abspath()` function to log the absolute paths. Even though `um2nc` does not change working directory, I usually prefer working with absolute paths in general.

**Comment by truth-quark** [comment]
In the interest of simplicity, I recommend going with a minimal implementation, deferring to Python defaults etc. Each specialisation, additional code branching etc contributes to more complexity, more testing and so forth. The less code we have, the easier it is for different devs to understand & maintain :-)

**Comment by atteggiani** [comment]
Yes that makes sense. 
Well, for now we can remove it, then if we need a way to work with absolute paths we will reinstate it.

However, if we remove it, we neeed a function to check existence of files and directories to replace the logic of the `check` and `checkdir` input arguments to `get_abspath()`.
